packages/@aws-cdk/aws-lambda/lib/alias.ts

```diff
@@ -193,6 +201,39 @@ export class Alias extends QualifiedFunctionBase implements IAlias {
     });
   }
 
+  /**
+   * Enable autoscaling for the given lambda version. Requires that lambda version has provisioned concurrency.
+   * @param props The properties for autoscaling
+   */
+  public autoScaleProvisionedConcurrency(props: EnableScalingProps): IScalableVersionAttribute {
+    if (!this.provisionedConcurrency) {
+      throw new Error('Autoscaling is available for aliases with provisioned concurrency only');
+    }
+    if (this.scalableAlias) {
+      throw new Error('Autoscaling already enabled for this alias');
+    }
+    // Use a Service Linked Role
+    // https://docs.aws.amazon.com/autoscaling/application/userguide/application-auto-scaling-service-linked-roles.html
+    const role = iam.Role.fromRoleArn(this, 'ScalingRole', Stack.of(this).formatArn({
+      service: 'iam',
+      region: '',
+      resource: 'role/aws-service-role/lambda.application-autoscaling.amazonaws.com',
+      resourceName: 'AWSServiceRoleForApplicationAutoScaling_LambdaConcurrency',
+    }));
+
+    this.scalableAlias = new ScalableVersionAttribute(this, 'VersionScaling', {
```

Comment:
VersionScaling should be renamed to AliasScaling since it's under the alias.

Reply:
Thanks, I'll be more careful with copy/paste in the future :)